MvvmCross-Plugins/PictureChooser/MvvmCross.Plugins.PictureChooser.Wpf/MvxPictureChooserTask.cs

```diff
@@ -86,12 +119,20 @@ private BitmapSource ConvertBitmapInBitmapSource(Bitmap bitmap)
                 new Rectangle(0, 0, bitmap.Width, bitmap.Height),
                 System.Drawing.Imaging.ImageLockMode.ReadOnly, bitmap.PixelFormat);
 
+            BitmapPalette palette = null;
+            System.Windows.Media.Color[] pal = null;
```

Comment:
Add a using to `System.Windows.Media`, no need to repeat this all over the place.